### ISSUE TITLE
An "Always stay on top" option

### ISSUE THREAD

**Issue by imnotpoz** [ThoNohT/NohBoard#156]
Well yeah that's it, I just think it would be cool if you added an option, so that until you minimise / close the window, it stays on top.

**Comment by ThoNohT** [comment]
If you close or minimize it, even an always on top window will close or minimize.
If you just use window or game capture on the window while it is in the background, it will still work.

The only way I could see this being useful is if you want to actually have it on your own screen over another program.

**Comment by imnotpoz** [comment]
Yep, that's what I had in mind, I was streaming typeracer to some friends over discord the other day and I thought about this. It would be really useful in that situation.

**Comment by Bulli** [comment]
I use TurboTop for this: https://www.savardsoftware.com/downloads.asp
My usecase is bugreport. I'd like to use Geforce Experience to save a video **after** something happens. For better understanding I'd like to include NohBoard in the video.

**Comment by HLEBNY** [comment]
@ThoNohT I guess it's easy to make. It needs TopMost parameter or something

**Comment by The-furf-of-July** [comment]
Can confirm this is a feature that would be useful for me as well.
